src/services/files/classifiers/google/document_ai/ReadMe.md

```diff
@@ -0,0 +1 @@
+https://cloud.google.com/document-ai/#classify-documents
```

Comment:
Just adding placeholders for other, 3rd party classifiers for now.
No time to experiment with these.